File: domain/Reaccionables/Interes.py
from datetime import datetime
from typing import List

from domain.Reaccionables.Rasgo import Rasgo
from domain.Reaccionables.Reaccion import Reaccion


class Interes(Rasgo):
    def __init__(self, reacciones: List[Reaccion], texto: str, fecha_modificacion: datetime) -> None:
        super().__init__(reacciones, texto, fecha_modificacion)

File: src/main/java/entities/usuarios/reaccionables/rasgos/Interes.java
package entities.usuarios.reaccionables.rasgos;

import entities.usuarios.reaccionables.Reaccion;

import javax.persistence.Entity;
import java.time.LocalDateTime;
import java.util.List;
@Entity
public class Interes extends Rasgo {

    public Interes(Topico topico, LocalDateTime fechaModificacion, List<Reaccion> reacciones) {
        super(topico, fechaModificacion, reacciones);
    }

    public Interes(){}
}
